nsqadmin/http.go

```diff
@@ -83,29 +87,29 @@ func NewHTTPServer(ctx *Context) *httpServer {
 		router:   router,
 	}
 
-	router.Handle("GET", "/ping", s.pingHandler)
-
-	router.Handle("GET", "/", s.indexHandler)
-	router.Handle("GET", "/nodes", s.nodesHandler)
-	router.Handle("GET", "/node/:node", s.nodeHandler)
-	router.Handle("GET", "/topic/:topic", s.topicHandler)
-	router.Handle("GET", "/topic/:topic/:channel", s.channelHandler)
-	router.Handle("GET", "/static/:asset", s.embeddedAssetHandler)
-	router.Handle("GET", "/counter", s.counterHandler)
-	router.Handle("GET", "/counter/data", s.counterDataHandler)
-	router.Handle("GET", "/lookup", s.lookupHandler)
-	router.Handle("GET", "/graphite_data", s.graphiteDataHandler)
-
-	router.Handle("POST", "/tombstone_topic_producer", s.tombstoneTopicProducerHandler)
-	router.Handle("POST", "/empty_topic", s.emptyTopicHandler)
-	router.Handle("POST", "/delete_topic", s.deleteTopicHandler)
-	router.Handle("POST", "/pause_topic", s.pauseTopicHandler)
-	router.Handle("POST", "/unpause_topic", s.pauseTopicHandler)
-	router.Handle("POST", "/empty_channel", s.emptyChannelHandler)
-	router.Handle("POST", "/delete_channel", s.deleteChannelHandler)
-	router.Handle("POST", "/pause_channel", s.pauseChannelHandler)
-	router.Handle("POST", "/unpause_channel", s.pauseChannelHandler)
-	router.Handle("POST", "/create_topic_channel", s.createTopicChannelHandler)
+	router.Handle("GET", "/ping", http_api.Decorate(s.pingHandler, log, http_api.PlainText))
+
+	router.Handle("GET", "/", http_api.Decorate(s.indexHandler, log))
+	router.Handle("GET", "/nodes", http_api.Decorate(s.nodesHandler, log))
+	router.Handle("GET", "/node/:node", http_api.Decorate(s.nodeHandler, log))
+	router.Handle("GET", "/topic/:topic", http_api.Decorate(s.topicHandler, log))
+	router.Handle("GET", "/topic/:topic/:channel", http_api.Decorate(s.channelHandler, log))
+	router.Handle("GET", "/static/:asset", http_api.Decorate(s.embeddedAssetHandler, log))
```

Comment:
@mreiferson turns out this doesn't work because log doesn't handle the response leaving nothing to render the response from the handler (ie: `s.embededAssetHandler`) into the response.

The effect is that requests for static assets return a 0 length content.